source/ada/lsp-ada_documents.adb

```diff
@@ -2837,8 +2838,8 @@ package body LSP.Ada_Documents is
 
       Filter : LSP.Ada_Completions.Filters.Filter;
    begin
-      if Parent.Is_Null
-        or else (Parent.Kind not in
+      if not Parent.Is_Null
```

Comment:
you removed the case where we do nothing when Parent.Is_Null => is this wanted?

Reply:
Yes, because it can happen in the case of use-clause completion. So I want to call the completion providers in this case, and let them handle the "Parent = null" case themselves (I have added a proper exception handler also, to prevent crashing the whole completion in case one provider fails).